my_account.rb

```diff
@@ -19,12 +19,12 @@
   #loans = Prototypes::Loans.new
   #session[:uniqname] = 'mrio' #need to get this from cosign?
   #patron = Patron.for(uniqname: session[:uniqname])
-  #erb :home, :locals => {loans: loans, patron: patron} 
```

Comment:
@niquerio Is there a way to grab the active session patron data? I can update the "Wecome, [name]" part, then.